docs/docs/usage.md

```diff
@@ -130,4 +130,28 @@ An example of the contents of the resulting `deptry.json` file is as follows:
     "transitive": [],
     "misplaced_dev": []
 }
-```
\ No newline at end of file
+```
+
+## usage in pre-commit
+
+_deptry_ can be added to your [pre-commit](https://pre-commit.com/) rules.  Here is
+an example config for your .pre-commit-config.yaml file:
+
+```
+-   repo: https://github.com/fpgmaas/deptry.git
+    rev: <tag>
+    hooks:
+    - id: deptry
+      args:
+        - "--skip-missing"
+```
+
+Replace <tag> with one of the [tags](https://github.com/fpgmaas/deptry/tags) from the
```

Comment:
Suggestion to replace the `NOTE:` with an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types)  using the `!!! info` syntax.

Also, maybe we should state more explicitly that the pre-commit hook should be run within the virtual environment, and we could replace the word `virtualenv` with `virtual environment`.

```
!!! important

    This will only pull in the pre commit hooks config file from the version you have specified.  The actual version of deptry that will be run will be the first one found in your path, so you will need to add deptry to your local virtual environment for the pre-commit.
    
    For the pre-commit hook to run successfully, it should be run within the virtual environment of the project to be scanned, since it needs access to the metadata of the installed packages.
   
```